generators/ci-cd/index.js

```diff
@@ -159,7 +159,7 @@ module.exports = class extends BaseGenerator {
             this.template('.gitlab-ci.yml.ejs', '.gitlab-ci.yml');
         }
         if (this.pipeline === 'circle') {
-            this.template('circle.yml.ejs', 'circle.yml');
+            this.template('config.yml.ejs', '.circleci/config.yml');
```

Comment:
You should keep the source as circle.yml, config.yml is confusing.

Reply:
@mshima : Unfortunately that's not possible as far as I know. Circle CI 2.0 wants it to be `.circleci/config.yml` just as Circle CI 1.0 wants it to be `circle.yml`. I suppose they included the `circle` part in the containing directory rather than the file name itself. 

https://circleci.com/docs/2.0/configuration-reference/

Reply:
I don't get, the dest is `.circleci/config.yml`, keep only the source as `circle.yml.ejs`.

Reply:
Ah, got what you mean. Thanks. Done. 😄